packages/storage/src/indexed-db/index.ts

```diff
@@ -292,21 +293,13 @@ export class IndexedDb implements IndexedDbInterface {
     );
   }
 
-  async initGasPrices() {
-    const savedGasPrices = await this.getGasPrices();
-    // These are arbitrarily set, but can take on any value.
-    // The gas prices set here will determine the fees to use Penumbra.
-    //
-    // Note: this is a temporary measure to enable gas prices in the web, but once
-    // https://github.com/penumbra-zone/penumbra/issues/4306 is merged, we can remove this.
-    if (!savedGasPrices) {
-      await this.saveGasPrices({
-        verificationPrice: 1n,
-        executionPrice: 1n,
-        blockSpacePrice: 1n,
-        compactBlockSpacePrice: 1n,
-      });
-    }
```

Comment:
can you extrapolate? `GasPrices` has an `AssetId` field.

Reply:
You are right, `GasPrices` does contain an `asset_id`, I missed that because `asset_id` is never defined. 
But the problem is that our `GasPrices` storage is deprecated and not suitable for multi-asset fees

Now we store `GasPrices` assuming that we can only have one record 
```
GAS_PRICES: {
    key: 'gas_prices';
    value: Jsonified<GasPrices>;
  };
``` 
Instead, we should use the `assetId` as the key
```
GAS_PRICES: {
    key: Jsonified<Required<GasPrices>['assetId']['inner']>;
    value: Jsonified<GasPrices>;
  };
``` 
But in that case, I expect we'll run into an error, because pd purposely omits the assetId for native `GasPrices`, but that's bad for us, because undefined can't be a key in indexed-db

We will have to manually add the `assetId` for native `GasPrices` or come up with another trick

Also, note that the getGasPrices function will also have to be modified 
```typescript
async getGasPrices(): Promise<GasPrices | undefined> {
    const jsonGasPrices = await this.db.get('GAS_PRICES', 'gas_prices');
    if (!jsonGasPrices) return undefined;
    return GasPrices.fromJson(jsonGasPrices);
  }
```  

Also need to change the **_gasPrices rpc_** for the view service so that it retrieves `alt_gas_prices`

And the block processor logic that should save `alt_gas_prices` to indexed-db when needed

Reply:
It might make sense to consider this as a separate PR afterwards (but keep in mind that alternative fees won't work correctly)

Reply:
tracking in https://github.com/penumbra-zone/web/issues/1310